config/interface.go

```diff
@@ -7,11 +7,18 @@ import (
 )
 
 type VendorConfigManager interface {
-	EnableTPM()
-	EnableSRIOV()
-
 	Raw(name, value string, menuPath []string)
 	Marshal() (string, error)
+	Unmarshal(cfgData string) (err error)
+	StandardConfig() (biosConfig map[string]string, err error)
+
+	BootMode(mode string) error
```

Comment:
Would recommend smaller interfaces, so not all vendor implementations are forced to implement all of these